src/sbus/request/sbus_request_hash.c

```diff
@@ -150,6 +151,11 @@ sbus_requests_add(hash_table_t *table,
     item->req = req;
     item->conn = conn;
     item->is_dbus = is_dbus;
+    item->member = talloc_strdup(item, member);
```

Comment:
Is it safe to call `talloc_strdup()` if `member == NULL`?

Reply:
Yes, it will just return NULL.

```
_PUBLIC_ char *talloc_strdup(const void *t, const char *p)
{
	if (unlikely(!p)) return NULL;
	return __talloc_strlendup(t, p, strlen(p));
}
```